extensions/amp-story/amp-story-auto-ads.md

```diff
@@ -75,45 +72,28 @@ looks like the following:
  `amp-ad` element to be inserted.
 
 The above example will insert the following `amp-ad` element, which represents
-a [custom ad](../../ads/custom.md):
+a [ad served by doubleclick](../extensions/amp-ad-doubleclick-impl/amp-ad-network-doubleclick-internal.md):
 
 ```html
-<amp-ad type="custom"
-  data-url="https://adserver.com/getad?slot=abcd1234"
+<amp-ad type="doubleclick"
+  data-slot="/30497360/a4a/amp_story_dfp_example"
 </amp-ad>
 ```
 
 Unlike normal `amp-ad`, no `<fallback>` or `<placeholder>` needs to be specified
 here, as ads in story can only be displayed once fully rendered.
 
-## Inlined templates
-If [custom ad](../../ads/custom.md) is used, templates can be inlined in the
-`<amp-story-auto-ads>` element:
+## Validation
 
-```html
-  <amp-story-auto-ads>
-    <script type="application/json">
-        {
-          "ad-attributes": {
-            "type": "custom",
-            "data-url": "https://adserver.com/getad?slot=abcd1234"
-          }
-        }
-    </script>
+It has to be a direct child of `amp-story` element.
 
-    <template type="amp-mustache" id="template-1">
-      <amp-img layout="fill" src="{{imgSrc}}"></amp-img>
-      <amp-pixel src="{{impressionUrl}}"></amp-pixel>
-    </template>
+## Insertion Control
+The AMP runtime will determine the best place to insert an ad using our own heuristics.
+If there is a specific position in a story that you wish to never show an ad, you
```

Comment:
nit: I think this is worded weird (at least to me). Can we do something like: 


> If there is a specific position in a story that you wish to never show an ad, you can add the `next-page-no-ad` attribute to the `<amp-story-page>`. The insertion algorithm will then ignore the page following these `<amp-story-page>` elements as possible places to place an ad.

Reply:
If that sounds too redundant, feel free to just ignore this.

Reply:
made changes.